### PR TITLE
first simple event API

### PR DESCRIPTION
**Pull request by pavlobaron**
Bypasses events to the locally running agent using  https://instana.atlassian.net/wiki/display/DOCS/Event+SDK+REST+Web+Service

### CODE REVIEW

File: event.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	ServicePlugin = "com.instana.forge.connection.http.logical.LogicalWebApp"
+	ServiceHost   = ""
+)
+
+func SendDefaultServiceEvent(title string, text string, severity int) {
+	SendServiceEvent(sensor.serviceName, title, text, severity)
+}
+
+func SendServiceEvent(service string, title string, text string, severity int) {
+	sendEvent(&EventData{
```
Comment (boyand) [inline event.go]: Do we need to care about agent state here? I.e. agent.sensor.canSend() or this API is agnostic

Comment (pglombardo) [inline event.go]: This is agnostic.  Can send events at anytime regardless if process is announced or not.  We may want to test error handling eventually though (in the case of no host agent) and make sure things are handled graciously.

Comment (pavlobaron) [inline event.go]: the frequency with which events arrive is not expected to be as high as traces or metrics, plus events API on the agent is generic and doesn't need any dances around pid in different scenarios.

**Comment by pglombardo** [comment]
Tested locally - looks great 👍  thanks @pavlobaron

**Pull request merged**